### PR TITLE
Hammerdb: add redis complete state for VM  

### PR DESCRIPTION
**Pull request by ebattat**
### Description
add redis complete state for VM  
### Fixes
Add completed state for the following file:
db_creation_postgres_vm.yml
db_creation_mariadb_vm.yml
db_creation_mssql_vm.yml

### CODE REVIEW

File: docs/hammerdb.md [hunk 1]
```diff
@@ -5,18 +5,18 @@
...
 An example CR might look like this
 
 ```yaml
 apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
 kind: Benchmark
 metadata:
-  name: hammerdb
-  namespace: benchmark-operator
+  name: hammerdb-benchmark
+  namespace: my-ripsaw
```
Comment (rsevilla87) [inline docs/hammerdb.md:19]:
```suggestion
  namespace: benchmark-operator
```

**Review by rsevilla87 — COMMENTED** [review]
I don't understand why are we uploading so many **examples** of the hammerdb benchmark. We should provide some samples of how to using the benchmark, however it's on the final user to code and and warehouse the CRs, this is not the right place to store all test scenarios.

**Comment by ebattat** [comment]
Hi,

I dont know why it was added to my PR (maybe because using existing branch
name)
My PR should contains only:
https://github.com/cloud-bulldozer/benchmark-operator/pull/629/commits/9109e61b30bb5241ae831ffdb4c52c790c759f00

Best regards,
Eli Battat.

**Comment by ebattat** [comment]
We create a new one only with redis changes

**Pull request closed without merging**